MAUI/TextInputLayout/Custom-Icons.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: Custom Icons in .NET MAUI Text Input Layout control | Syncfusion
+description: Learn here all about Custom Icons support in Syncfusion .NET MAUI Text Input Layout (SfTextInputLayout) control and more.
+platform: maui
+control: SfTextInputLayout
+documentation: ug
+---
+
+# Custom Icons in .NET MAUI Text Input Layout (SfTextInputLayout)
+
+Any custom icons can be added to the leading edge or the trailing edge of input view in the text input layout control. The events and commands related to the custom icons should be handled at the application level.
+
+Unicode or font icons for the labels can be displayed as icons.
+
+N> Refer to the following links to learn more about font icons:
+* [How to create font icons using our metro studio and export as ttf?](https://help.syncfusion.com/metro-studio/export-icon-font)
+* [How to set font family for the custom fonts in labels?](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/fonts?view=net-maui-7.0#display-font-icons)
+
+## Leading view
+
+A label can be added as a leading icon for the input view by setting the `LeadingView` property. It can be positioned either inside or outside the container by setting the `LeadingViewPosition` property. By default, it is positioned `Outside`.
```

Comment:
A label can be added as a leading icon for the input view by setting the `LeadingView` property. It can be positioned either inside or outside the container by setting the `LeadingViewPosition` property. By default, it is positioned `Outside.`

Reply:
Addressed the changes